docs/uberon-editor-sop.md

```diff
@@ -137,6 +137,37 @@ Example: “DG granule cell layer” would be an exact synonym of 'dentate gyrus
 
 Compare the previous example to “EC layer 1”, which should be a _related_ synonym (with synonym type "abbreviation") of 'entorhinal cortex layer 1', where there is not enough context to confidently determine what "EC" stands for.
 
+## Design Pattern Usage with DOSDP
+
+
+The Uberon ontology is composed of a large number of terms, classifications and relationships, which continue to increase. As maintaining manually all these classifications and relationships would be an arduous task, a substantial portion of the maintenance is automated. This automation depends, to a significant extent, on the systematic application of design patterns.
+
+Uberon uses **Dead simple OWL design patterns** (DOSDP, [Osumi-Sutherland *et al.*, 2017](https://doi.org/10.1186/s13326-017-0126-0)) to document simple patterns, as they require minimal programming expertise, and once implemented, it is easy to edit.
+
+All patterns are stored in [/src/patterns/dosdp-patterns](https://github.com/obophenotype/uberon/tree/master/src/patterns/dosdp-patterns), while the editable tables are located in [/src/patterns/data/default](https://github.com/obophenotype/uberon/tree/master/src/patterns/data/default).
+
+The purpose of this document is to describe the existing patterns to make them more accessible to the users.
```

Comment:
This sentence sets an expectation that all patterns in https://github.com/obophenotype/uberon/tree/master/src/patterns/dosdp-patterns will be described in this document.
Only one pattern is described below (vessel_connecting_branch_of_vessel_pattern.yaml), and that does not appear in the dosdp-patterns directory.